compute/vm_with_identity.go

```diff
@@ -10,6 +10,8 @@ import (
 	"fmt"
 
 	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-06-01/compute"
+	"github.com/Azure/azure-sdk-for-go/services/preview/msi/mgmt/2015-08-31-preview/msi"
+	"github.com/pkg/errors"
```

Comment:
dont we need this in the toml file ?

Reply:
This is part of the standard library.  It got pulled in from calls to ```errors.Wrap()```.  See [this](https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully) for more context.

Reply:
is it ? https://godoc.org/github.com/pkg/errors vs package errors ?

Reply:
I stand corrected.  I'll run ```dep ensure -update``` and push the changes.